app/src/main/java/com/futurice/freesound/network/api/model/SoundSearchResult.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2016 Futurice GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.futurice.freesound.network.api.model
+
+data class SoundSearchResult (
+    val count: Int,
+    val next: String?,
```

Comment:
I've already written that: https://github.com/peter-tackage/kotlin-options/blob/master/kotlin-options-moshi-adapter/src/main/kotlin/com/petertackage/kotlinoptions/moshi/MoshiJsonAdapters.kt